docs/UserGuide.md

```diff
@@ -673,6 +679,12 @@ Shortcut: `addbi t/TITLE a/AMOUNT [c/CATEGORY...]`
 
 * `CATEGORY` is optional. Multiple `c/` prefixes can be used to specify multiple categories.
 
+> :warning: &nbsp; Adding of duplicate bookmark incomes is not allowed.
+> Bookmark incomes with the same title are considered to be duplicates of each other.
+> Contiguous whitespace characters between words in the title will be treated as a single whitespace character.
+>
+> For example: `Part Time` and `Part  Time` are considered to be the same title while `Part Time` and `PartTime` are not considered to be the same title.
```

Comment:
Can you add non-breaking spaces to all example code snippets for both the warnings? I'm afraid this might give rise to ambiguity.

![image](https://user-images.githubusercontent.com/5585517/97776017-0e166c80-1ba0-11eb-8ced-dbe317e25bb1.png)